ckan/lib/helpers.py

```diff
@@ -148,12 +148,12 @@ def redirect_to(*args, **kw):
         import ckan.plugins.toolkit as toolkit
 
         # Redirect to /dataset/my_dataset.
-        toolkit.redirect_to(controller='package', action='read',
+        toolkit.redirect_to('dataset.read',
```

Comment:
This is the example on how to build a Pylons-based URL :) Change it to a controller still not migrated or just use placeholders like "my_controller" and "my_action"